docs/pkglists/Rocky_8.x86_64

```diff
@@ -0,0 +1,73 @@
+debug/x86_64/bareos-bconsole-debuginfo
+debug/x86_64/bareos-common-debuginfo
```

Comment:
I think we don't need this file, and we can probably also remove CentOS_8.x86_64.

Reply:
Good point

Reply:
I have removed these files.